src/components/upload-form/upload-slide.js

```diff
@@ -82,11 +89,17 @@ class UploadSlide extends Component {
                 type="range"
                 min="0"
                 max="10"
-                defaultValue="0"
+                value={retentionYears}
+                onChange={slider => {
+                  selectRetentionYears(slider.target.value);
+                  this.setState({
+                    storageCost: ((this.state.fileSize / BYTES_IN_GIGABYTE) * slider.target.value).toFixed(8)
+                  });
+                }}
               />
             </div>
             <div className="upload-column">
-              <select id="sel" disabled>
+              <select id="sel" disabled value={retentionYears}>
```

Comment:
>disabled  

Thinking it might be time to pull that out.

Reply:
okay, i wasnt sure if it was intended for that to be a slider change only

Reply:
Just so that I'm clear, removing that "disabled" will enable the slider, correct?  We can take that out now.

Reply:
The slider is already enabled, that will enable the number dropdown box.